leroybm/fp-manager
# Add static metadata object to root layout

Next.js's Metadata API introduce enhanced application metadata management within the project's Rootlayout component. This update focuses on optimizing SEO and web shareability by utilizing the API to define meta and link tags in the HTML head element.

## Changes Made
- Integrated Next.js's Metadata API into the Rootlayout component.

## How to Test
- Verify that the Next.js application builds successfully after the changes.
- Inspect the HTML head element for the presence of relevant meta and link tags generated by the Metadata API.
- Validate the defined meta tags like the ones in the example below.

<img width="1049" alt="nextjs-add-metadata" src="https://github.com/leroybm/fp-manager/assets/13338262/2651aaad-204f-4f8b-a5d5-4875c465eca3">
